docs/plugins/tv.md

```diff
@@ -38,34 +38,34 @@ During _development_ the `channel`-method returns a random mocked channel. Optio
 }
 ```
 
-#### Change current channel
+#### Change Current Channel
 
 ```js
 const channelNumber = 2
 TV.channel(channelNumber).then(channel => console.log(channel))
 ```
 
-When a channelNumber is passed as an argument to the `channel`-method it will try to change the TV channel.
-It will return a _promise_ which returns the channel information as an object.
+If a `channelNumber` is passed as an argument, the `channel` method attempts to change the TV channel.
+It returns a *Promise* that returns the channel information as an object.
```

Comment:
Maybe we should add that the promise is rejected in case of an error (or if the app is now allowed to set the channel)

Reply:
Added: "The Promise is *rejected* if an error occurred or if the App is not allowed to set the channel."